birdset/modules/models/LTU.py

```diff
@@ -0,0 +1,164 @@
+import torch
+
+from typing import Dict, Optional, Tuple
+import warnings
+
+import datasets
+import pandas as pd
+import tensorflow as tf
+import tensorflow_hub as hub
+import torch
+from torch import nn
+from ...utils.label_utils import get_label_to_class_mapping_from_metadata
+
+from .embedding_abstract import EmbeddingModel
+
+class LTU(nn.Module):
+    """
+    
+    Attributes:
+        PERCH_TF_HUB_URL (str): URL to the TensorFlow Hub model for bird vocalization.
```

Comment:
Fix docs, replace perch with LTU.

Reply:
I didn't use this model I think it may have come from merging BEATS into this branch